pkg/storage/mock/storer.go

```diff
@@ -41,6 +43,18 @@ func WithSubscribePullChunks(chs ...storage.Descriptor) Option {
 	})
 }
 
+func WithBaseAddress(a swarm.Address) Option {
```

Comment:
This was removed, This is not necessary anymore since we use the push sync mock, this was code to emulate what we had in Swarm.
But It was changed accordingly,
Thanks for the correction.

Reply:
If somebody needs this, we can change it in another PR.